packages/dev/core/src/Audio/sound.ts

```diff
@@ -431,6 +435,15 @@ export class Sound {
                 this._connectedTransformNode.unregisterAfterWorldMatrixUpdate(this._registerFunc);
                 this._connectedTransformNode = null;
             }
+
+            if (this._tryToPlayTimeout) {

```

Comment:
this part could be factorized in a function I guess as It seems to be there 3 times ?